editor/export/project_export.cpp

```diff
@@ -292,6 +300,21 @@ void ProjectExportDialog::_edit_preset(int p_index) {
 	exclude_filters->set_text(current->get_exclude_filter());
 	server_strip_message->set_visible(current->get_export_filter() == EditorExportPreset::EXPORT_CUSTOMIZED);
 
+	patches->clear();
+	TreeItem *patch_root = patches->create_item();
+	Vector<String> patch_list = current->get_patches();
+	for (int i = 0; i < patch_list.size(); i++) {
+		TreeItem *patch = patches->create_item(patch_root);
+		String patch_path = patch_list[i];
+		patch->set_cell_mode(0, TreeItem::CELL_MODE_STRING);
+		patch->set_editable(0, false);
```

Comment:
This was a sloppy change on my part in general. I noticed that the editing of the actual tree item was broken, and I figured the use-case for it was mildly redundant, since you can just press the edit button (and get the actual file dialog) instead, so I just disabled the editing altogether. This however seemingly broke the edit button as well, which I failed to catch.

So instead I went ahead and fixed the editing of the tree item and restored this to `set_editable(0, true)`.

Reply:
Is there any value in making it editable though? It displays and allows to edit only file name.
btw edited item does not update its tooltip.

Reply:
You mean as in removing the editing altogether, including the per-item edit/browse button?

If you feel it's not worth the code/clutter then I can remove it. I would maybe argue the drag-and-drop should be removed as well in that case. It's arguably even more niche than the ability to edit the items.

Reply:
No, I mean the editing name on double-click. It only edits the name, so it's useless if you e.g. moved the file elsewhere. You can just pick it again with the browse button.

Drag and drop is fine, some people will always want it even if order doesn't matter functionally.

Reply:
I'm on a different branch now so I can test this right now, but as mentioned above it seemed like the per-item edit/browse button broke as well when I disabled editing on the tree item, where the item wouldn't update after I selected a file from the edit/browse dialog, hence why I re-enabled it.

I didn't look into exactly why, since fixing the editing was straight-forward, but I can take another look tomorrow.

Reply:
Editing only affects being able to text edit the item, setting text will work regardless.